exercises/practice/resistor-color/resistor-color.sql

```diff
@@ -0,0 +1,5 @@
+-- Task one: Populate "colors" table with colors and index/value.
+-- Schema: CREATE TABLE "colors" ("index" INT, "color" TEXT);
+--
+-- Task two: Update "color_code" with the correct value/result based on the color.
+-- Schema: CREATE TABLE "color_code" ("color" TEXT, "result" INT);
```

Comment:
The descriptions of the tasks would be better (or, additionally) placed in `.docs/instructions.append.md`.

Reply:
Oh. Yeah. I/We should probably utilize those :D We may need them heavily for this track. Will add one!